src/meta/src/barrier/checkpoint/creating_job/barrier_control.rs

```diff
@@ -43,7 +44,7 @@ pub(super) struct CreatingStreamingJobBarrierControl {
     // key is prev_epoch of barrier
     inflight_barrier_queue: BTreeMap<u64, CreatingStreamingJobEpochState>,
     backfill_epoch: u64,
-    initial_epoch: Option<u64>,
+    is_first_committed: bool,
```

Comment:
nit: `is_first_committed` reads really similar to `is_first_commit`. However the usage is different. I think the intention is to track if the first epoch has been committed.

`first_epoch_committed` could be a better name.